ush/machine/cheyenne.yaml

```diff
@@ -17,18 +17,18 @@ platform:
   RUN_CMD_NEXUS: mpirun -np $nprocs
   RUN_CMD_AQMLBC: mpirun -np ${NUMTS}
   PRE_TASK_CMDS: '{ ulimit -s unlimited; ulimit -a; }'
-  TEST_EXTRN_MDL_SOURCE_BASEDIR: /glade/work/epicufsrt/contrib/UFS_SRW_data/develop/input_model_data
-  TEST_AQM_INPUT_BASEDIR: /glade/work/epicufsrt/contrib/UFS_SRW_data/develop/aqm_data
-  TEST_PREGEN_BASEDIR: /glade/work/epicufsrt/contrib/UFS_SRW_data/develop/FV3LAM_pregen
-  TEST_ALT_EXTRN_MDL_SYSBASEDIR_ICS: /glade/work/epicufsrt/contrib/UFS_SRW_data/develop/dummy_FV3GFS_sys_dir
-  TEST_ALT_EXTRN_MDL_SYSBASEDIR_LBCS: /glade/work/epicufsrt/contrib/UFS_SRW_data/develop/dummy_FV3GFS_sys_dir
-  TEST_VX_FCST_INPUT_BASEDIR: '{{ "/glade/work/epicufsrt/contrib/UFS_SRW_data/develop/output_data/fcst_" }}{{ "ens" if (global.NUM_ENS_MEMBERS > 0) else "det" }}{{ "/{{workflow.PREDEF_GRID_NAME}}" }}{% raw %}{% endraw %}'
-  FIXaer: /glade/work/epicufsrt/contrib/UFS_SRW_data/develop/fix/fix_aer
-  FIXgsm: /glade/work/epicufsrt/contrib/UFS_SRW_data/develop/fix/fix_am
-  FIXlut: /glade/work/epicufsrt/contrib/UFS_SRW_data/develop/fix/fix_lut
-  FIXorg: /glade/work/epicufsrt/contrib/UFS_SRW_data/develop/fix/fix_orog
-  FIXsfc: /glade/work/epicufsrt/contrib/UFS_SRW_data/develop/fix/fix_sfc_climo
-  FIXshp: /glade/work/epicufsrt/contrib/UFS_SRW_data/develop/NaturalEarth
+  TEST_EXTRN_MDL_SOURCE_BASEDIR: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/input_model_data
+  TEST_AQM_INPUT_BASEDIR: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/aqm_data
+  TEST_PREGEN_BASEDIR: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/FV3LAM_pregen
+  TEST_ALT_EXTRN_MDL_SYSBASEDIR_ICS: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/dummy_FV3GFS_sys_dir
+  TEST_ALT_EXTRN_MDL_SYSBASEDIR_LBCS: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/dummy_FV3GFS_sys_dir
+  TEST_VX_FCST_INPUT_BASEDIR: '{{ "/glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/output_data/fcst_" }}{{ "ens" if (global.NUM_ENS_MEMBERS > 0) else "det" }}{{ "/{{workflow.PREDEF_GRID_NAME}}" }}{% raw %}{% endraw %}'
+  FIXaer: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/fix/fix_aer
+  FIXgsm: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/fix/fix_am
+  FIXlut: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/fix/fix_lut
+  FIXorg: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/fix/fix_orog
+  FIXsfc: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/fix/fix_sfc_climo
+  FIXshp: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/NaturalEarth
 data:
```

Comment:
Paths for `ics_lbcs` could be updateduse the pre-staged data on Cheyenne/Derecho in 
/glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/input_model_data (as in ./ush/machine/derecho.yaml)

```data:
  ics_lbcs:
    FV3GFS: 
      nemsio: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/input_model_data/FV3GFS/nemsio/${yyyymmdd}${hh}
      grib2: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/input_model_data/FV3GFS/grib2/${yyyymmdd}${hh}
      netcdf: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/input_model_data/FV3GFS/netcdf/${yyyymmdd}${hh}
    NAM: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/input_model_data/NAM/${yyyymmdd}${hh}
    HRRR: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/input_model_data/HRRR/${yyyymmdd}${hh}
    RAP: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/input_model_data/RAP/${yyyymmdd}${hh}
    GSMGFS: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/input_model_data/GSMGFS/${yyyymmdd}${hh}
    ```